src/League/Configurator.php

```diff
@@ -10,33 +10,18 @@
 
 final class Configurator implements ContainerConfigurator
 {
-    /**
-     * @var ServiceProviderInterface[]
-     */
-    private $providers = [];
-
-    public function addApplicationConfig(ApplicationConfig $config, $prefix = 'config')
-    {
-        $this->providers[] = new ApplicationConfigServiceProvider($config, $prefix);
-    }
-
-    public function addServiceConfig(ServiceConfig $config)
+    public function addApplicationConfig($container, ApplicationConfig $config, $prefix = 'config')
     {
-        $this->providers[] = new ServiceServiceProvider($config);
+        $container->addServiceProvider(new ApplicationConfigServiceProvider($config, $prefix));
     }
 
-    public function addInflectorConfig(InflectorConfig $config)
+    public function addServiceConfig($container, ServiceConfig $config)
     {
-        $this->providers[] = new InflectorServiceProvider($config);
+        $container->addServiceProvider(new ServiceServiceProvider($config));
     }
 
-    /**
-     * @internal
-     *
-     * @return ServiceProviderInterface
```

Comment:
@tomphp 

Can we also remove the now unused import for `League\Container\ServiceProvider\ServiceProviderInterface`?